src/components/IIIFPlayer/IIIFPlayer.md

```diff
@@ -6,7 +6,8 @@ IIIFPlayer component, provides a wrapper consisting of the Context providers con
 
 ** __Either `manifestUrl` or `manifest` is REQUIRED. If both props are given then `manifest` takes *precedence* over `manifestUrl`__
 
-- `customErrorMessage`: accepts a messagge to display to the user in the unlikely event of the component crashing, this has a default error message and it is _not required_. The message can include HTML markup.
+- `customErrorMessage`: accepts a messagge to display to the user in the unlikely event of the component crashing, this has a default error message and it is _not required_. The message can include HTML markup. This prop has default value for a generic message and it is _not required_ to initialize the component.
```

Comment:
I like the consistency between this and the new `emptyManifestMessage` description but a bit got repeated.  What do you think about this revision?
```suggestion
- `customErrorMessage`: accepts a message to display to the user in the unlikely event of the component crashing. The message can include HTML markup. This prop has default value for a generic message and it is _not required_ to initialize the component.
```